packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "@okta/courage": "4.4.0",
+    "@okta/courage": "4.6.0-beta.2050.g7ec8178",
```

Comment:
can we use a release version? Generally speaking, we've been using only release versions for the sign-in widget

Reply:
discussed offline, we are good with beta version